src/routes/anime/index.tsx

```
@@ -0,0 +1,166 @@
+	CarouselItem,
+	CarouselNext,
+	CarouselPrevious,
+} from "@/components/ui/carousel.tsx";
+import animesData from "@/lib/mockups/animes.json";
+
+export const Route = createFileRoute("/anime/")({
+	component: animeRoute,
+});
+
+function animeRoute() {
+	const { t } = useTranslation();
```

Biome (2.4.4)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)